tcl/org.testeditor.tcl.model/model/tcl.xcore

```diff
@@ -128,9 +128,8 @@ class TestStepWithAssignment extends TestStep {
 	contains AssignmentVariable variable opposite testStep
 }
 
-class AssignmentVariable {
+class AssignmentVariable extends VariableReference{
```

Comment:
no. there is no need for the assignment variable in itself. variable reference should do.